ecl/hqlcpp/hqlttcpp.cpp

```diff
@@ -1778,6 +1778,20 @@ static IHqlExpression * simplifySortlistComplexity(IHqlExpression * sortlist)
                     appendComponent(cpts, invert, &concats.item(idxc));
             }
         }
+        else if (cur->getOperator() == no_trim)
+        {
+            //TRIM(fixed-length-string) can just sort by the string instead.  (Don't match LEFT/RIGHT versions.)
```

Comment:
Is it really true? What happens if a string contains a control character less than ascii(32)?